app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,38 @@
+class CommentsController < ApplicationController
+  before_action :authenticate_user!, only: [:create, :destroy]
+    
+  # POST "/microposts/20/comments"
+  def create
+    micropost = Micropost.find(params[:micropost_id])
+    if micropost
+      comment = micropost.comments.build(comment_params)
+      comment.user = current_user
+    
+      if comment.save
+        flash[:success] = t('comment_posted')
+        micropost.user.notifications.create(sender_id: current_user.id, notification_type: :comment)
```

Comment:
can move this whole logic to a method in some model so that notification always gets created whenever a comment is posted. Moreover, it will follow DRY principle.